modules/accredited_representative_portal/app/services/accredited_representative_portal/saved_claim_service/create.rb

```diff
@@ -28,6 +28,7 @@ def perform(
             form_id = saved_claim.class::PROPER_FORM_ID
             organize_attachments!(form_id, attachment_guids).tap do |attachments|
               saved_claim.form_attachment = attachments[:form]
+              saved_claim.user_account_id = user_account_id
```

Comment:
@patrick-brown-oddball Would it be helpful to add defensive programming here? Maybe a check to see of the user_account_id exists.